src/game/core/explore.js

```diff
@@ -11,24 +10,27 @@ import {
 
 import { Observable } from 'rx'
 
-import models from '../models'
+import {
+  maps,
+  monsters,
+} from '../models'
+
 import { run } from './combat'
 
-export function randomMonster (mapId) {
-  const mapObj = models.maps.find(mapId)
-  const monsterPool = mapObj.monsters.reduce((acc, monster) => [
+export function randomMonster (monsters, pickMonsterFn=monsters.find) {
```

Comment:
Taking account we are currently implementing a linting system, `pickMonsterFn=monsters.find` will break futurely. `pickMonsterFn=monsters.find` `->` `pickMonsterFn = monsters.find`.